### ISSUE TITLE
OpenEMR 7.0.2 on Debian 12.6 Bookworm php setting detection bug

### ISSUE THREAD

**Issue by apmuthu** [openemr/openemr#7603]
### Describe the bug

OpenEMR 7.0.2 on Debian 12.6 Bookworm **Install Step 4** detection of **php.ini** setting **max_execution_time** is already at **120** but is detected as **0** and states it should be **at least 60**

### To Reproduce
![OpenEMP702_Step4InstallBug](https://github.com/user-attachments/assets/488eae99-a740-4689-926f-8db2f5e30f97)

Normal install

### Expected behavior
Should show 120 and not be an error

### Client configuration

- Browser: Firefox 128.0.3 64bit
- OpenEMR version: 7.0.2
- Operating system: Debian Linux 12.6 Bookworm - Server side

**Comment by apmuthu** [comment]
There is no way to re-check the php.ini settings after setting the php.ini file and restarting the webserver on Step 4 of the Install page.

**Comment by apmuthu** [comment]
![OpenEMR702_after_desired_php ini_settings](https://github.com/user-attachments/assets/3f14855f-7495-4ad3-bbac-25b4e209e772)

**Comment by adunsulag** [comment]
@apmuthu Have you verified your apache server restarted?  If you create a phpinfo.php file in your root installation with the following contents:
```
<?php phpinfo():
?>
```

And look at your server variables you can double check your web server has the correct max_execution_time setting.  If you can do this and let us know that your php.ini is indeed not being read properly we can mark this as a bug.

BTW in case you didn't know max_execution_time of 0 actually means the script can run forever which is a valid way to run OpenEMR it just means you could potentially have some scripts that never die until the webserver is reset.

**Comment by apmuthu** [comment]
Yes I have restarted the webserver. The second screenshot shows the changed variables are in current column. The current column for the said variable should take the value from the **php.ini** file.
